embedded-kafka/src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala

```diff
@@ -236,6 +236,7 @@ sealed trait EmbeddedKafkaSupport {
     props.put("group.id", s"embedded-kafka-spec")
     props.put("bootstrap.servers", s"localhost:${config.kafkaPort}")
     props.put("auto.offset.reset", "earliest")
+    props.put("enable.auto.commit", "false")
```

Comment:
This was probably overlooked by my side as I didn't realise that. I'll be fine with leaving `false` as default, but it's still better to add a parameter to enable or disable this.